Xamarin.Essentials/TextToSpeech/TextToSpeech.android.cs

```diff
@@ -0,0 +1,290 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Android.OS;
+using Android.Speech.Tts;
+using Xamarin.Essentials;
+
+namespace Xamarin.Essentials
+{
+    public static partial class TextToSpeech
+    {
+        const int maxSpeechInputLengthDefault = 4000;
+
+        static global::Android.Speech.Tts.TextToSpeech textToSpeech;
+
+        static TextToSpeech()
+        {
+            Task.Run(() => Initialize());
+
+            return;
+        }
+
+        static Java.Util.Locale localedefault = null;
+
+        static async Task Initialize()
+        {
+            // set up the TextToSpeech object
+            // third parameter is the speech engine to use
+            textToSpeech = new global::Android.Speech.Tts.TextToSpeech(Platform.CurrentContext, default(TextToSpeechListener), "com.google.android.tts");
+
+            var locales = await GetLocalesAsync();
+
+            return;
+        }
+
+        public static async Task SpeakAsync(string text, CancellationToken cancelToken = default(CancellationToken)) =>
+             await SpeakAsync(text, default(SpeakSettings), cancelToken);
+
+        public static async Task SpeakAsync(string text, SpeakSettings settings, CancellationToken cancelToken = default(CancellationToken))
+        {
+            if (string.IsNullOrEmpty(text))
+            {
+                throw new ArgumentNullException(nameof(text), "Text cannot be null or empty string");
+            }
+
+            Task initialization = null;
+            if (textToSpeech == null)
+            {
+                initialization = Initialize();
+            }
+
+            var parts = SplitText(text, maxSpeechInputLengthDefault);
+
+            await Task.Run(() =>
```

Comment:
Fixing initizlied in the actual listener means you can just await on it. Don't do a new Task.Run